Sources/TuistKit/Commands/InitCommand.swift

```diff
@@ -110,9 +107,10 @@ extension InitCommand {
             }
             .flatMap { $0 }
 
+        let templatesWithoutValidation = ["default", "swiftui"]
```

Comment:
I'm not sure I understand why we need this special case, what would happen if we don't add this?

Reply:
If we don't add this — with this PR running either the `tuist init -t swiftui` or `tuist init --template swiftui` will start failing with the following error:
> Error: Missing expected argument '--name <name>'

Such behavior would be different from the current one, where the swiftui template can be initialized without specifying the name.

Reply:
understood 👍 
Given we are releasing a 3.0, I think we are fine changing the behavior and requiring a name, WDYT?

Tagging also @fortmarek in case he has anything to add 🚀

Reply:
@danyf90 I just had a thought that probably we should mark this change as breaking since custom templates now will require specifying the name.
Should I add additional entry into the CHANGELOG or just move the `fix` one into the `breaking` section?

Reply:
Why require a name when it's not actually necessary?

Reply:
@Austinate I would add it to the `Changed` section

@ezraberch you will use `tuist init` very few times, I don't see a big issue in asking the user which name to use instead of assuming to use the folder name